README.md

```diff
@@ -8,7 +8,7 @@ Content to include:
 * Marketplace link
 * A reference to the version of xenia tested on
 * A short description of how the game runs or any issues
-* A log file (run xenia with "xenia default.xex > output.log" and upload output.log to [gist.github.com](https://gist.github.com/))
+* A log file (run xenia with "xenia defaulted - - log_file=output.log" and upload output.log to [gist.github.com](https://gist.github.com/))
```

Comment:
not sure where the other changes came from - it should be `xenia default.xex --log_file=output.log`